src/utils/utils-vega.js

```diff
@@ -108,7 +109,7 @@ export function createVegaAttrMixin(
         const colorScaleName = layerName + "_" + attrName,
           colorsToUse = layerObj.defaultFillColor(),
           domainInterval = 100 / (colorsToUse.length - 1),
-          linearScale = colorsToUse.map((color, i) => i * domainInterval / 100),
+          linearColorScale = colorsToUse.map((color, i) => i * domainInterval / 100),
```

Comment:
Here I just renamed a "linearScale" to be more specific, to not just let it be shadowed by the new one above